tendrl/commons/message.py

```diff
@@ -108,6 +150,8 @@ def serialize_message(obj):
     elif isinstance(obj, datetime.datetime):
         serial = obj.isoformat()
         return serial
+    elif isinstance(obj, Exception):
```

Comment:
@shtripat we are string {"exception": ex} in payload so ex is an exception object, problem is object in json gives serialization problem, so at the dumping an object this code will called automatically and check any object is there. if any object of Exception type then it is dumped as yaml. this only for serialization problem